ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/workers-rhel/provision/cucushift-installer-rehearse-azure-ipi-workers-rhel-provision-chain.yaml

```diff
@@ -0,0 +1,31 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-workers-rhel-provision
```

Comment:
Maybe we can rename the chain and workflow name to replace `workers-rhel` with `rhel8`?